requirements.txt

```diff
@@ -19,7 +19,6 @@ coverage==5.3
 cryptography==2.5
 deepdiff==6.2.3
 Dickens==2.0.0
-Django==3.2.12
```

Comment:
I don't understand why previous sorting is bad

Reply:
Updated commit description:
```
Issues with current sorting:
    - Sorting is inconsistent due to case sensitivity.
      For example, `Django` is listed below `django-*` packages,
      while it should precede them in a case-insensitive alphabetical order
    - Proper alphabetical sorting requires ignoring case differences,
      ensuring Django appears before `django-*`
```
      
There is currently no documented standard for sorting `requirements.txt`, which leads to inconsistencies. To ensure uniformity, we should use a single tool or command
Maybe I missed on how to sort `requirements.txt`

Probably should create issue if we don't have a consistent way to sort `requirements.txt`?